app/controllers/admin/workshops_controller.rb

```diff
@@ -162,4 +170,20 @@ def workshop_created_within_specific_time_frame?
     Time.zone.now.between?(@workshop.created_at,
                            @workshop.created_at + WORKSHOP_DELETION_TIME_FRAME_SINCE_CREATION)
   end
+
+  def physical_workshop_and_no_host?(workshop)
+    !workshop.virtual && host_id.empty?
+  end
+
+  def no_host_error
```

Comment:
@despo this method is suuper confusing, especially in the way we are using it above. We are basically saying if there is a an error update the workshop and show a success message. I find that very confusing.

Reply:
Agreed. It's meant to read no (host error) which is definitely not obvious.